docker-compose.yaml

```diff
@@ -161,7 +161,7 @@ services:
     profiles: [ "full" ]
     volumes:
       - /var/log:/var/log
-    command: -config.file=/etc/promtail/config.yml
```

Comment:
@akihikokuroda I had doubts about this configuration file. I didn't find it and I don't know if it's correct to change it. I tested everything and seems to be working but I would like to know your opinion.

Reply:
I _believe_ that file is in the container (and we don't touch it), so I don't think we want to update that one

Reply:
Solved in [f553791](https://github.com/Qiskit-Extensions/quantum-serverless/pull/918/commits/f5537916109f0e02752817d8909ad120f837fdca)